Tutorial handouts (Tutorial 1-4)

Xintong-Li opened the pull request:
This pull request is about sections reg. tutorial handouts.

I made the following changes:

- I added a new einvironment called `tutorial`. This environment has distinct color (resembling https://github.com/siefkenj/MultivariableCalculus/raw/master/src/MultivariableCalculus.pdf).
- A new folder "tutorial" is created to save all tutorial-related files. 
- For each tutorial, there're three files: `tutorialX.tex` is the file containing student's handout; `tutorialX-solutiions.tex` is the file containing solutions to the problems; `tutorialX-instructions.tex` is the file containing instructive materials.
- At this moment, I only added to the main .tex file codes rendering student's handouts, leaving codes rendering solutions and instructions commented. Maybe a discussion is needed to decide for which version should we enable/disable codes for solutions and instructions. However, all tutorial-related materials for tutorial 4 are rendered to showcase the appearence of tutorial section.

siefkenj commented on book/linearalgebra.tex:
```diff
@@ -8901,5 +8908,77 @@ \section*{Eigenvectors}
+			These problems relate to the following course learning
+			objective: \textit{work independently to understand concepts and procedures that have not
+			been previously explained to you}.
+	\end{objectives}
```
Keep the learning objectives in the tutorial file, and leave things as `\section*`, etc.. You can keep `\Title` in this file. I'd like to have it so that the same sources are included for both the book and the stand-alone tutorials.

siefkenj commented on book/tutorials/tutorial1-solutions.tex:
```diff
@@ -0,0 +1,28 @@
+\Heading{Solutions}
```
No headings in these.

siefkenj commented on book/tutorials/tutorial1-instructions.tex:
```diff
@@ -0,0 +1,92 @@
+		\Heading{Learning Objectives} Students need to be able to\ldots
```
Leave this as `\section*`

Xintong-Li commented:
For tutorial 8, the macro `\rank` is not defined in the book. Thus, I changed all `\rank` to `\Rank`.

For tutorial 9, the macro `\autoaligne` is not defined in the book. (It seems that the tutorial file introduced the autoaligne package.) Thus, I temperily used array environment to recreate the system of equations.

Tutorial 8 has the context subsection but it has no content.

I have changed all `\Heading` in the previous commit to `\subsection*` to match the original source code.
